browser/resources/settings/brave_overrides/settings_subpage.ts

```diff
@@ -18,7 +18,7 @@ RegisterStyleOverride(
         background-color: var(--leo-color-page-background) !important;
         box-shadow: none !important;
     }
-    :host(:not([id=safetyHub])) slot {
+    :host(:not(.multi-card):not([id=safetyHub])) slot {
```

Comment:
Without this change, it seems multiple sections in subpage can't be separated as we want.